ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -144,23 +145,22 @@ public AccountDao(MySqlDataAccessor dataAccessor, MySqlAccountServiceConfig mySq
     insertDatasetSql =
         String.format("insert into %s (%s, %s, %s, %s, %s, %s, %s, %s) values (?, ?, ?, ?, now(3), ?, ?, ?)",
             DATASET_TABLE, ACCOUNT_ID, CONTAINER_ID, DATASET_NAME, VERSION_SCHEMA, LAST_MODIFIED_TIME, RETENTION_COUNT,
-            USER_TAGS, DELETE_TS);
+            RETENTION_TIME, USER_TAGS);
```

Comment:
From the ddl, I saw we added RETENTION_TIME but we still keep the DELETE_TS.
Why we don't deal with DELETE_TS here?

Reply:
How do we save the expiration time in the old scheme? I don't see it in the schema. Are we using the "DELETE_TS" for the expiration time?

Reply:
I plan to not support future ttl for dataset. so the retentionTime is the default retention value which all dataset versions under the dataset can inherited from. And the delete_ts is only used for dataset's lifespan, and the dataset can only be deleted by calling the deleteDataset API. Previously I tried to use the delete_ts to represent both default ttl and dataset's lifespan, but it's not make sense because if we create a dataset with 30days ttl, which means any dataset version upload request will be failed after the dataset creation time + 30 days.

Reply:
So DELETE_TS will be updated when deleteDataset is called, right?
When we insert the dataset entry, we don't provide the DELETE_TS. What's the default value for it, -1?

Reply:
That's right.